doc/versionHistory.md

```diff
@@ -1,5 +1,10 @@
 # Version History
 
+0.4.1
+
+- Add **integrationTest.vi** in `test/` directory.
```

Comment:
- Add **integrationTest.vi** in `test/` directory and related tests.

Reply:
Done